### PR TITLE
Compute: upgrade to package-2020-06-30

### PR DESCRIPTION
**Pull request by xccc-msft**

### CODE REVIEW

**Review by ChenTanyi — APPROVED** [review]

File: src/ResourceManagement/Compute/Generated/Models/DataDisk.cs [hunk 1]
```diff
@@ -174,7 +174,7 @@ public DataDisk(int lun, DiskCreateOptionTypes createOption, string name = defau
         /// Gets or sets the managed disk parameters.
         /// </summary>
         [JsonProperty(PropertyName = "managedDisk")]
-        public Management.ResourceManager.Fluent.SubResource ManagedDisk { get; set; }
+        public ManagedDiskParametersInner ManagedDisk { get; set; }
```
Comment (weidongxu-microsoft) [inline src/ResourceManagement/Compute/Generated/Models/DataDisk.cs]: What causes this change? What's in swagger?

**Review by weidongxu-microsoft — APPROVED** [review]

File: src/ResourceManagement/Compute/Generated/Models/DataDisk.cs (hunk 1 above)
Comment (xccc-msft) [reply src/ResourceManagement/Compute/Generated/Models/DataDisk.cs]: I think it's generator issue, already existing when we did upgrade last time [here](https://github.com/Azure/azure-libraries-for-net/pull/911/commits/bc14e74928c0e67c97b474da4cfafc2d87aafaf3)

File: Samples/Compute/ConvertVirtualMachineToManagedDisks.cs [hunk 2]
```diff
@@ -52,7 +52,7 @@ public static void RunSample(IAzure azure)
                             .WithNewVhd(50)
                             .WithLun(2)
                             .Attach()
-                        .WithSize(VirtualMachineSizeTypes.StandardD3V2)
+                        .WithSize(VirtualMachineSizeTypes.Parse("Standard_D2a_v4"))
```
Comment (weidongxu-microsoft) [inline Samples/Compute/ConvertVirtualMachineToManagedDisks.cs:55]: What's the cause?

File: Samples/BatchAI/ManageBatchAI.cs [hunk 3]
```diff
@@ -15,6 +15,7 @@
 using Microsoft.Azure.Management.Storage.Fluent.Models;
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.File;
+using ExecutionState = Microsoft.Azure.Management.BatchAI.Fluent.Models.ExecutionState;
```
Comment (weidongxu-microsoft) [inline Samples/BatchAI/ManageBatchAI.cs:18]: Why this change?

Comment (xccc-msft) [reply Samples/BatchAI/ManageBatchAI.cs:18]: Compute have same named `ExecutionState` after generate new version.

File: Samples/Compute/ConvertVirtualMachineToManagedDisks.cs (hunk 2 above)
Comment (xccc-msft) [reply Samples/Compute/ConvertVirtualMachineToManagedDisks.cs:55]: For test purpose, we want to move vm size into AMD based.
